app/forms/hyrax/default_work_form.rb

```diff
@@ -4,14 +4,18 @@ module Hyrax
   class DefaultWorkForm < Hyrax::Forms::WorkForm
     include ::ScholarsArchive::TriplePoweredProperties::TriplePoweredForm
     self.model_class = ::DefaultWork
-    self.terms += [:resource_type, :date_available, :date_copyright, :date_issued, :date_collected, :date_valid, :date_accepted, :replaces]
+    self.terms += [:doi,:alt_title , :license, :based_near, :resource_type, :date_available, :date_copyright, :date_issued, :date_collected, :date_valid, :date_accepted, :replaces] + [:hydrologic_unit_code, :funding_body, :funding_statement, :in_series, :tableofcontents, :bibliographic_citation, :peerreviewed, :additional_information, :digitization_spec, :file_extent, :file_format, :dspace_community, :dspace_collection]
```

Comment:
Can you fix the spacing around :alt_title and the comma? Also, can't the `+` be removed and the arrays combined since all fields fall under `+=` ?